src/main.cpp

```diff
@@ -1220,10 +1220,10 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState &state, const C
 }
 
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit, bool fRejectAbsurdFee)
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit, const CAmount nAbsurdFee)
```

Comment:
FWIW, I didn't know that const-by-value could be ever considered worse than regular by-value either.
@laanwj can you extend on this or were you just pointing at the fact that you can always turn a const-by-value into a const-by-ref unless you want to avoid the warning you'll get with basic types?